src/warnings.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as chalk from 'chalk';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import {SourceRange} from './ast/source-range';
+import {Severity, Warning} from './editor-service';
+
+export enum Verbosity {
```

Comment:
Typescript doesn't support enums like that. We could make it a union of string literals though, like: 

```typescript
export type Verbosity = 'one-line' | 'full';
```

OTOH, this works in JS with the enum: 

```javascript
Verbosity.OneLine 
// => 0
Verbosity[0]
// => 'OneLine'
```

I'm kinda leaning to the former. What do you think @justinfagnani ?